src/main/java/com/venafi/vcert/sdk/utils/FeignUtils.java

```diff
@@ -22,18 +22,27 @@
 import feign.slf4j.Slf4jLogger;
 import com.venafi.vcert.sdk.Config;
 import com.venafi.vcert.sdk.connectors.tpp.Tpp;
+import org.apache.http.conn.ssl.TrustSelfSignedStrategy;
+import org.apache.http.ssl.SSLContexts;
+
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLSocketFactory;
 
 public class FeignUtils {
 
   static Supplier<GsonBuilder> gsonBuilderFactory = GsonBuilder::new;
 
+  //ALWAYS SET THIS FLAG TO FALSE BEFORE COMMIT.
+  //This flag allows to bypass SSL verification in local dev environments.
+  private static final boolean DEBUG = false;
```

Comment:
Moved the debug flag to an environment variable. When not present, the value is set to false. This will be documented along the other environment variables used for testing purposes